hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieParquetConfig.java

```diff
@@ -37,7 +37,7 @@
 
   public HoodieParquetConfig(T writeSupport, CompressionCodecName compressionCodecName, int blockSize,
                              int pageSize, long maxFileSize, Configuration hadoopConf, double compressionRatio) {
-    this(writeSupport, compressionCodecName, blockSize, pageSize, maxFileSize, hadoopConf, compressionRatio, false);
+    this(writeSupport, compressionCodecName, blockSize, pageSize, maxFileSize, hadoopConf, compressionRatio, true);
```

Comment:
can we do HoodieStorageConfig.PARQUET_DICTIONARY_ENABLED.defaultValue

Reply:
Discussed offline: there's no reason for this value not to be configured. Will follow-up and rebase all users of this ctor to instead rely on the configured value